gen/build_deploy/bash.py

```diff
@@ -558,13 +558,25 @@ def validate_custom_check_bins_dir(custom_check_bins_dir):
     return $OVERALL_RC
 }
 
+function setup_exhibitor_tls_bootstrap()
+{
+    read -d '' ca_data << EOF || true
```

Comment:
Do we want variable expansion on the contents of the heredoc? We should use quoted `'EOF'` to prevent variable expansion.

Reply:
we do not, that would be bad.